sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -611,7 +676,34 @@ internal static LinkedEntityMatch LinkedEntityMatch(double confidenceScore = def
                 recognizePiiEntitiesActionResults.ToList(),
                 recognizeLinkedEntitiesActionsResults.ToList(),
                 analyzeSentimentActionsResults.ToList(),
-                null); // TODO: include extractive summarization API (https://github.com/Azure/azure-sdk-for-net/issues/22836).
+                new List<ExtractSummaryActionResult>());
```

Comment:
Updated this to empty list instead of null. Empty list seems more reasonable because that's what we would have if no Extract Summary Actions were sent.